client/src/pages/SignUp/GeneralInfo/GeneralInfo.tsx

```diff
@@ -14,6 +14,10 @@ function GeneralInfo(): JSX.Element {
 
   // Function to handle button click
   const handleClick = () => {
+    if (!setFirstName || !setLastName || !setUserName || !setEmail || !setPhone || !setDoB) {
```

Comment:
I don't think this logic is correct. I'm in merge conflict hell right now trying to merge my changes for #67, and while testing this doesn't work. I think you wanted something like
```ts
    if (!firstName || !lastName || !userName || !email || !phone || !DoB) {
```
I'm working on merging my PR, so I can fix it for you when I merge if you want.

Reply:
Yes please do fix it. I did it this way after removing those "unused" variables for the last vercel deployement error I had. But if it deploys with that fix I'm all for it!